### PR TITLE
Better warning message for delegation

### PR DESCRIPTION
**Pull request by anderson-joyle**
To support DV issue https://github.com/microsoft/Power-Fx-Dataverse/issues/183.

### CODE REVIEW

**Comment by LucGenetier** [comment]
✅ No public API change.

**Review by MikeStall — DISMISSED** [review]

File: src/strings/PowerFxResources.en-US.resx [hunk 1]
```diff
@@ -5139,7 +5139,7 @@
     <comment>Error Message.</comment>
   </data>
   <data name="WrnDelagationTableNotSupported" xml:space="preserve">
-    <value>Delegating this operation on table '{0}' is not supported.</value>
```
Comment (MikeStall) [inline src/strings/PowerFxResources.en-US.resx:5142]: We're losing the table name? That may be useful to show

File: src/strings/PowerFxResources.en-US.resx [hunk 2]
```diff
@@ -5139,7 +5139,7 @@
...
+    <value>This operation may not work as expected on large tables (>{0} rows). We are working on delegating formulas to support large data sets.</value>
```
Comment (MikeStall) [inline src/strings/PowerFxResources.en-US.resx:5142]: "We are working on delegating formulas to support large data sets" - wording here probably needs to be tweaked. 

What error does Power Apps show in this case?

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/strings/PowerFxResources.en-US.resx [hunk 3]
```diff
@@ -5139,7 +5139,7 @@
     <comment>Error Message.</comment>
   </data>
   <data name="WrnDelagationTableNotSupported" xml:space="preserve">
```
Comment (CarlosFigueiraMSFT) [inline src/strings/PowerFxResources.en-US.resx:5141]: >WrnDelagationTableNotSupported [](http://example.com/codeflow?start=14&length=30)

Where is this used? If we are changing the string from expecting one argument to expecting two, we need to pass the additional argument to it.

**Review by MikeStall — DISMISSED** [review]

File: src/strings/PowerFxResources.en-US.resx (hunk 3 above)
Comment (anderson-joyle) [reply src/strings/PowerFxResources.en-US.resx:5141]: This is going to be used from Dataverse project side. Please check PR https://github.com/microsoft/Power-Fx-Dataverse/pull/187.

Comment (CarlosFigueiraMSFT) [reply src/strings/PowerFxResources.en-US.resx:5141]: We shouldn't have a resource from this project that is not used here. This should be moved to the PowerFx-Dataverse project instead.

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/strings/PowerFxResources.en-US.resx [hunk 4]
```diff
@@ -5139,7 +5139,11 @@
...
-    <value>Delegating this operation on table '{0}' is not supported.</value>
+    <value>Delegation warning. Table '{0}' part might not work correctly.</value>
```
Comment (CarlosFigueiraMSFT) [inline src/strings/PowerFxResources.en-US.resx:5142]: >Delegation warning [](http://example.com/codeflow?start=11&length=18)

The error type for this error is already a warning - we probably don't need to say it again in the message.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Review by MikeStall — APPROVED** [review]

**Pull request merged**